src/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* .star-section {
+  display: flex;
+  width: 140px;
+  height: 160px;
+  padding: 32px 22px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+} */
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+  margin-right: 4px;
+}
+
+.stars {
+  list-style: none;
+  width: 96px;
```

Comment:
The styles are almost the same as for container. You shouldn't set the fixed sizes for it. Set the sizes and margins for stars elements